app/(client)/profile/page.tsx

```diff
@@ -0,0 +1,224 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { ChevronLeft, User } from 'lucide-react';
+
+import FacultySelect from '@/components/auth/selects/FacultySelect';
+import GroupSelect from '@/components/auth/selects/GroupSelect';
+import { Button } from '@/components/ui/button';
+import { Form } from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { useToast } from '@/components/ui/toast/use-toast';
+import useAuth from '@/hooks/useAuth';
+import { api } from '@/lib/api';
+import { useProfileStore } from '@/store/profile-store';
+import { UserI } from '@/types/auth/user.interface';
+import { IFacultyData } from '@/types/faculty.interface';
+import { IGroupData } from '@/types/group.interface';
+
+import { changeDataSchema, ChangeProfileFormData } from './_validation';
+
+const PersonalData = () => {
+  const [selectedFaculty, setSelectedFaculty] = useState<IFacultyData>();
+  const [selectedGroup, setSelectedGroup] = useState<IGroupData>();
+  const { userData, setUserData, isProfileMenuActive, setIsProfileMenuActive } =
+    useProfileStore(state => state);
+  const { toast } = useToast();
+  const { loggedIn } = useAuth();
+
+  const form = useForm<ChangeProfileFormData>({
+    resolver: zodResolver(changeDataSchema),
+  });
+
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors, isSubmitting },
+  } = form;
+
+  useEffect(() => {
+    if (!loggedIn) return;
+    const fetchUser = async () => {
+      try {
+        await api.get<UserI>('/user').then(res => {
```

Comment:
You don't need to use both await and .then() in the same code block because they serve the same purpose